tests/e2e/p/interchain_workflow.go

```diff
@@ -95,14 +95,8 @@ var _ = e2e.DescribePChain("[Interchain Workflow]", ginkgo.Label(e2e.UsesCChainL
 		nodeID, nodePOP, err := infoClient.GetNodeID(e2e.DefaultContext())
 		require.NoError(err)
 
-		// Validation and delegation durations don't actually matter to this
-		// test - it is only ensuring that adding a validator and delegator
-		// doesn't break interchain transfer.
-		var (
-			validatorEndTime = time.Now().Add(30 * time.Second)
-			delegatorEndTime = validatorEndTime.Add(-1 * time.Second)
```

Comment:
this is a leftover from a previous iteration where duration instead of end time was specified. It's actually better to keep the same endTimes now